### DISCUSSION TITLE
Netbeans ARM64

### DISCUSSION THREAD

**Discussion by nobukts** [apache/netbeans#3811] [Q&A]
Hi, i just wanna know if it is a way to install Netbeans in a chromebook with arm64 architecture, just that.

Thanks in advance for all of you.

**Comment by Chris2011** [comment]
I googled and find some entries for NetBeans on a raspberry pi. So I think this is possible. Maybe you should need to download the zip archive. But official we don't have an entry for that in our Wiki/Docu afaik.

**Comment by neilcsmith-net** [comment]
Try the any architecture .deb under the community packages on the downloads page. I use that on my Chromebook (x86_64) and Pi. Should work for you. The Arm32 AppImage on the same page might work too.

This all assuming you're using Crostini?

**Reply by lkishalmi** [reply]
The zip distribution should work. Some native bindings might be missing (like profiler).

**Comment by BradWalker** [comment]
I don't know if anyone has answered your question.

But, it does seem to me to be as simple as: 1 - make sure you have Java on
your computer of choice and 2 - use it to execute Netbeans.

I know the team does strive to make things execute on cross-platforms.
Having said that, we can't test everything.

If you do find an issue/problem, we would certainly be interested in
hearing about it though.

-brad w.
